client/components/Ratings.jsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import Rating from './Rating.jsx';
+
+function Ratings(props) {
+  // if the averages have NOT been calculated before && we have data from DB, then:
+  if(!props.foundAverage && props.reviews.length > 0) {  
+    const numReviews = props.reviews.length;
```

Comment:
As a note, you could also do this as: let accuracy, communication, cleanliness, location, checkIn, value;
accuracy = communication = cleanliness = location= checkIn = value = 0!

Reply:
I didn't realize this. It'll make more code more readable and concise to do this, for sure.